tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/GradleModuleProjectImporter.java

```diff
@@ -37,49 +37,67 @@ public IStatus canImport( String location )
 
         File file = new File( location );
 
-        if( file.exists() )
+        if( findGradleFile( file ) && findSettingsFile( file ) )
+        {
+            retval = Status.OK_STATUS;
+        }
+        else
         {
-            if( findGradleFile( file ) )
-            {
-                retval = Status.OK_STATUS;
-            }
-
             File parent = file.getParentFile();
 
             while( parent != null )
             {
-                if( findGradleFile( parent ) )
+                if( findGradleFile( parent ) && findSettingsFile( file ) )
                 {
```

Comment:
So go ahead and send me back the pull once you have tested it and made sure I didn't break anything else.